website/docs/node/node-configuration.mdx

```diff
@@ -0,0 +1,373 @@
+---
+id: node-config
+title: Node Configuration
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
```

Comment:
feel like we should add an introduction at the beginning here to:

1. tell people what config file CKB uses. take reference from https://github.com/nervosnetwork/ckb/blob/develop/docs/configure.md
3. add a bullet point for the 4 configs that the article is talking about so users can quickly jump to the interesting one.